src/main/webapp/form.html

```diff
@@ -20,8 +20,17 @@ <h3>a recommendation system for students, by students!</h3>
     <form id="user-form" method="POST" enctype="multipart/form-data">
       <label for="Name">Hello! my name is</label>
       <input type="text" name="name" class="name" required></input>
-      <label for="Department">I am currently a student in the department of</label>
+      <label for="Department">I am a student in the department of</label>
       <input type="text" name="department" class="department" required></input>
+      <label for="year">currently in year</label>
+      <select name="year">
+        <option value="1">1</option>
+        <option value="2">2</option>
+        <option value="3">3</option>
+        <option value="4">4</option>
+      </select>
+      <label for="phone">my phone number is</label>
+      <input type="tel" id="phone" name="phone" required></input>
```

Comment:
Shall we use [the pattern parameter](https://www.w3schools.com/tags/att_input_type_tel.asp) here to standardize the format?

Reply:
This is important, especially when we are getting the telephone number as a string. I tried it but I am afraid it might be too strict, e.g. in the example referred we have to add hyphen and cannot input mobile number. I searched it up and there is no general convention for phone number as the convention differs by the area or country. What we can do is, we can restrict it to UK mobile phone number which has 11 digits.

Reply:
sounds good then! Thanks for looking into it :)